internal/grpc/services/appprovider/appprovider.go

```diff
@@ -257,8 +258,35 @@ func (s *service) OpenFileInAppProvider(ctx context.Context, req *providerpb.Ope
 		appProviderURL += "?"
 	}
 	appProviderURL = fmt.Sprintf("%sWOPISrc=%s", appProviderURL, openResBody)
-	log.Info().Msg(fmt.Sprintf("Returning app provider URL %s", appProviderURL))
 
+	// In case of applications served by the WOPI bridge, resolve the URL and go to the app
+	// Note that URL matching is performed via string matching, not via IP resolution: may need to fix this
```

Comment:
This is ready for review, and I explicitly bring the attention to this note ;-) Feedback welcome.